usagov/vote-gov-drupal
# VOTE-1598 update state subtitles to remove state name

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1598

## Description (optional)
Remove state name from state page subtitles.

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. `lando retune`

### Testing steps
1. Visit http://vote-gov.lndo.site/register/ak and confirm that the state name has been removed
2. Switch to every translation available to make sure the state name is removed, except Navajo and Yup'ik.
